app/components/MixerVolmeter.vue.ts

```diff
@@ -149,7 +282,12 @@ export default class MixerVolmeter extends Vue {
   subscribeVolmeter() {
     this.volmeterSubscription = this.audioSource.subscribeVolmeter(volmeter => {
       this.setChannelCount(volmeter.peak.length);
-      this.drawVolmeter(volmeter.peak);
+
+      if (this.gl) {
+        this.drawVolmeterWebgl(volmeter.peak);
+      } else {
+        this.drawVolmeterC2d(volmeter.peak);
+      }
```

Comment:
If we pay attention on performance in this PR, we should draw volmeters on `requestAnimationFrame` event. It allows skipping some frames if CPU is overloaded

Reply:
I don't think that helps us here.  `requestAnimationFrame` executes the callback before the next paint.  In an animation loop using `requestAnimationFrame`, you would queue up your next `requestAnimationFrame` from within your callback.  This means that effectively your your callback interval matches your paint cycle.  As that slows, so does your interval.

However, simply adding a call in this callback to `requestAnimationFrame` does nothing for us, because the interval is a set 40ms by OBS.  Our callback will be called every 40ms by OBS regardless of the paint speed.  `requestAnimationFrame` would not save us any work because we would be using it incorrectly.

A potential solution could be to separate the OBS callback and the render callback.  So we simply store the latest data on the frontend, and then asynchronously run a traditional `requestAnimationFrame` loop that picks up that data and draws it with WebGL.  However I'm skeptical that would actual cause any tangible benefit in CPU usage.